frontend/src/components/UsersSelect/UsersSelect.tsx

```diff
@@ -4,25 +4,28 @@ import { ListItemIcon, ListItemText } from '@material-ui/core'
 import { ListItemLocation } from '../ListItemLocation'
 import { colors } from '../../styling'
 import { Icon } from '../Icon'
+import { getUsersConnectedDevice } from '../../models/devices'
 
 type Props = {
-  service: IService
+  device: IDevice
+  service?: IService
 }
 
-export const UsersSelect: React.FC<Props> = ({ service }) => {
+export const UsersSelect: React.FC<Props> = ({ device, service }) => {
   const location = useLocation()
-  const connected = service.sessions.length
+
+  const connected = service ? service.sessions.length : getUsersConnectedDevice(device).length
 
   return (
-    <ListItemLocation disabled={!service.access.length} pathname={location.pathname + '/users'}>
+    <ListItemLocation pathname={location.pathname + '/users'}>
       <ListItemIcon>
         <Icon name="user-friends" color={connected ? 'primary' : undefined} size="md" type="light" />
       </ListItemIcon>
       <ListItemText
-        primary="Users"
+        primary="Shared users"
         secondary={
           <>
-            {service.access.length ? service.access.length + ' total' : 'None'}
```

Comment:
I think if this is to correctly work for both device or services it needs to be passes as a single parameter, as it is now the users per service is going to show all users for the device instead.